asmrun/spacetime.c

```diff
@@ -568,7 +580,8 @@ static c_node* allocate_c_node(void)
 
   node->gc_header =
     Make_header(sizeof(c_node)/sizeof(uintnat) - 1, C_node_tag, Caml_black);
-  node->data.callee_node = Val_unit;
+  node->data.call.callee_node = Val_unit;
+  node->data.call.call_count = Val_long(0);
```

Comment:
I think this should probably stay unguarded, since there is a named struct field at this point.